sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -53,6 +53,15 @@ static const struct snd_soc_acpi_adr_device rt711_1_adr[] = {
 	}
 };
 
+static const struct snd_soc_acpi_adr_device workaround_rt711_0_adr[] = {
+	{
+		.adr = 0x000010025D070100, /* using broken _ADR as mapping to rt711 */
+		.num_endpoints = 1,
```

Comment:
override ?

Reply:
yes, I didn't touch tgl for now, this is needed as well.

Reply:
fixed now